src/Components/ui/Input/index.tsx

```diff
@@ -6,9 +6,17 @@ type InputType = InputProps & {
   inputWrapProps?: React.HTMLAttributes<HTMLDivElement>
   prefix?: any
   suffix?: any
+  isNumber?:boolean
 }
 
 export const Input = (props: InputType) => {
+  const isNumber = props?.isNumber || false;
+  const inputProps = isNumber ? {
+    type: 'number',
+    pattern: '[0-9]*',
+    inputMode: 'numeric',
+    placeholder: 'Nhập số'
```

Comment:
English only please. And do we need the default `placeholder` here?

Reply:
I will remove `placeholder` code